info/src/main/java/com/popup/info/controller/CrawlerController.java

```diff
@@ -0,0 +1,133 @@
+package com.popup.info.controller;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Locale;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import org.springframework.stereotype.Service;
+
+// import com.popup.info.domain.PopUpStoreInfo;
+import com.popupstore.info.PopUpStoreInfo;
+import com.popup.info.repository.InfoRepository;
+import com.popup.info.service.WebDriverManager;
+
+@Service
+public class CrawlerController {
+
+	// 크롤링할 대상 url (가장 상위 페이지)
+	private static final String MAIN_PAGE_URL = "https://www.popply.co.kr/popup?area=all";
+	private final WebDriver driver = WebDriverManager.initChromeDriver();
+	private HashMap<String, PopUpStoreInfo> popUpStoreInfos = new HashMap<>();
+	private HashMap<String, String> detailPageUrls = new HashMap<>();
+
+	@Autowired
+	private InfoRepository infoRepo;
+
+	// 0. 크롤링 시작
+	public void startCrawling() {
+		setCrawler();
+		getMainPageInfos();
+	}
```

Comment:
```suggestion
	@PostConstruct
	public void startCrawling() {
		setCrawler();
		getMainPageInfos();
	}
```
spring application을 시작하고 한 번만 실행하고 싶은 메소드 위에 `@PostConstruct`를 사용하면 프로그램 안정성이 올라가요!